### PR TITLE
Simple web api

### PR DESCRIPTION
**Pull request by johnsall**
#### 📲 What

This repositorey now hosts only the simple web api version of the project. You can find the previous full version of stacks in the stacks-java-cqrs-events repository. There is also the stacks-java-cqrs repo containig a version with CQRS but not messaging. Additionally, functional test have been removed from this project and the packages have been migrated from com.xxAMIDOxx.xxSTACKSxx to com.amido.stacks. We can now use Maven archetypes to dynamically rename them and any relevant files.

#### 🤔 Why

GitHub doesn't make builds required when you have a single repo with several builds triggered only when their specific folder is changes (AzDevOps does it). Because of that, the idea of a monorepo with several projects was changed to 3 repos, each one being one option on the CLI. This change is also mirrored in the .net project as well. 

#### 🛠 How

Any CQRS or messaging (ASB) relevant code and dependencies have been removed and test have been updated accordingly. 

#### 🕵️ How to test

In the java folder run:
./mvnw archetype:create-from-project
cd target/generated-sources/archetype
..\..\..\mvnw install
<cd to a test directory where the project can be created in>
mvn archetype:generate -DarchetypeGroupId='com.amido' -DarchetypeArtifactId='stacks-api-cqrs-archetype' -DarchetypeVersion='1.0.0-SNAPSHOT' -DgroupId='com.test' -DartifactId='stacks-api' -Dversion='1.0.0-SNAPSHOT' -Dpackage='com.test.stacks'

The generated project should now be under the com.test.stacks package name.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ X] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

**Comment by ElvenSpellmaker** [comment]
Hey, what's happening with the functional tests? Are we no longer doing post-deploy tests for the base repo? How can we ensure the repo works if not?

**Comment by ElvenSpellmaker** [comment]
> 
> 
> > Hey, what's happening with the functional tests? Are we no longer doing post-deploy tests for the base repo? How can we ensure the repo works if not?
> 
> We're doing integration tests confirming the API is working properly, but as there is no logic left in the code, just endpoint returning mock data, there isn't anything to functionally test.

I believe we still need a few Functional post-deploy tests to ensure the app is up and running correctly. Otherwise the pods could be fine and pass a rollout status check but the Ingress could be broken and we can't get to the pods.

**Comment by johnsall** [comment]
> 
> 
> > > Hey, what's happening with the functional tests? Are we no longer doing post-deploy tests for the base repo? How can we ensure the repo works if not?
> > 
> > 
> > We're doing integration tests confirming the API is working properly, but as there is no logic left in the code, just endpoint returning mock data, there isn't anything to functionally test.
> 
> I believe we still need a few Functional post-deploy tests to ensure the app is up and running correctly. Otherwise the pods could be fine and pass a rollout status check but the Ingress could be broken and we can't get to the pods.

The some tests for status check only, i.e. hitting the health check endpoint?

**Comment by ElvenSpellmaker** [comment]
> The some tests for status check only, i.e. hitting the health check endpoint?
Yeah that might be OK. At least something that says, hey I can get to the API and it's alive sort of thing.

**Comment by johnsall** [comment]
> 
> 
> > The some tests for status check only, i.e. hitting the health check endpoint?
> 
> Yeah that might be OK. At least something that says, hey I can get to the API and it's alive sort of thing.

Cool, I'll re add them then

**Comment by johnsall** [comment]
re-added functional tests to check for connectivity of the app

**Comment by ElvenSpellmaker** [comment]
Nice one, thanks @johnsall

File: api-tests-karate/src/test/java/components/menu/ReadApi.feature [hunk 1]
```diff
@@ -5,30 +5,4 @@ Feature: Read API
...
-      | 'f91d2f8c-35cc-45dd-11b1-11ca548e1111' |
-
-  Scenario: Search menu by page size and page number
-    Given url base_url.concat(menu)
-    And header Authorization = auth.bearer_token
-    And param pageSize = 1
-    And param pageNumber = 2
-    When method GET
-    Then status 200
+    Then status 200
```
Comment (ElvenSpellmaker) [inline api-tests-karate/src/test/java/components/menu/ReadApi.feature:8]: Missing end newline

File: java/src/main/resources/application.yml [hunk 2]
```diff
@@ -45,9 +41,4 @@ azure:
     uri: https://amido-stacks-tmp.vault.azure.net/
     client-id: xxxxxx
     client-key: xxxxxx
-    tenant-id: xxxxxx
-  servicebus:
-    connectionString: xxxxxx
-    topicName: menu-updates
-    subscriptionName: Subscription1
-    enabled: false
+    tenant-id: xxxxxx
```
Comment (ElvenSpellmaker) [inline java/src/main/resources/application.yml:44]: Missing newline

**Comment by johnsall** [comment]
Why are the remaining tests pending?

**Comment by ElvenSpellmaker** [comment]
Oh you need to run Pipelines for Stacks manually now, as it was a security risk having them auto run (due to Pipelines as Code). You can tell the bot to run by commenting `/azp run`

**Pull request merged**